samples/MvvmSample.Core/MvvmSample.Core.csproj

```diff
@@ -0,0 +1,14 @@
+﻿<Project Sdk="MSBuild.Sdk.Extras">
+
+  <PropertyGroup>
+	  <TargetFrameworks>netstandard2.0</TargetFrameworks>
+	  <LangVersion>8.0</LangVersion>
+  </PropertyGroup>
+	<ItemGroup>
+    <PackageReference Include="Microsoft.Toolkit.Mvvm" Version="7.0.0-build.156" />
+    <PackageReference Include="Newtonsoft.Json" Version="12.0.3" />
```

Comment:
Is there a specific reason we're using Newtonsoft.Json here instead of `System.Text.Json`? @Sergio0694 was this part of the original?

Reply:
We're using `System.Text.Json`, yes, though unfortunately the `refit` dependency still also pulls in `Newtonsoft.Json`, so we have that assembly too. It shouldn't be listed explicitly though, not sure why that was added here. See:

https://github.com/windows-toolkit/MVVM-Samples/blob/2277a9511836b0044f59b411f84132a382b263a0/samples/MvvmSampleUwp/MvvmSampleUwp/MvvmSampleUwp.csproj#L283-L285